pkg/bgp/extended-community.go

```diff
@@ -109,6 +118,12 @@ func makeExtCommunity(b []byte) (*ExtCommunity, error) {
 		ext.SubType = &st
 		l = 6
 		p++
+	case 3: 
+		st := uint8(b[p])
+                ext.SubType = &st
+		l = 6
+		p+=3
```

Comment:
is only 4 last bytes used for color community?

Reply:
correct, first 2 bytes are reserved, although following draft will use 2 bits (co-flags) : ietf-spring-segment-routing-policy ... to be added later